libs/go.json

```diff
@@ -94,15 +94,18 @@
   {"id":"go.cobra","imports":["github.com/spf13/cobra"],"name":"Cobra","repo":"spf13/cobra","tags":["cli-apps"],"tech":["cli"],"status":"awaiting-model"},
   {"id":"go.pty","imports":["github.com/kr/pty"],"name":"pty","repo":"kr/pty","tags":["unix-terminals"],"tech":["cli"],"status":"awaiting-model"},
   {"id":"go.spinner","imports":["github.com/briandowns/spinner"],"name":"Spinner","repo":"briandowns/spinner","tags":["progress-indicator"],"tech":["cli"],"status":"model-ready"},
+  {"id":"go.termui","imports":["github.com/gizak/termui"],"name":"termui","repo":"gizak/termui","tags":["dashboard"],"tech":["cli"],"status":"awaiting-model"},
   {"id":"go.aws-sdk","imports":["github.com/aws/aws-sdk-go"],"name":"AWS SDK","repo":"aws/aws-sdk-go","tags":["api"],"tech":["cloud"],"status":"model-ready"},
   {"id":"go.distribution","imports":["github.com/docker/distribution"],"name":"Distribution","repo":"docker/distribution","tags":["docker-distribution"],"tech":["cloud"],"status":"awaiting-model"},
   {"id":"go.docker","imports":["github.com/docker/docker"],"name":"docker","repo":"docker/docker","tags":["docker"],"tech":["cloud"],"status":"awaiting-model"},
   {"id":"go.gizmo","imports":["github.com/NYTimes/gizmo"],"name":"Gizmo","repo":"NYTimes/gizmo","tags":["microservices"],"tech":["cloud"],"status":"model-ready"},
+  {"id":"go.go-micro","imports":["github.com/micro/go-micro"],"name":"go-micro","repo":"micro/go-micro","tags":["microservices"],"tech":["cloud"],"status":"awaiting-model"},
   {"id":"go.goa","imports":["github.com/goadesign/goa"],"name":"goa","repo":"goadesign/goa","tags":["microservices"],"tech":["cloud"],"status":"model-ready"},
   {"id":"go.goamz","imports":["github.com/mitchellh/goamz"],"name":"goamz","repo":"mitchellh/goamz","tags":["aws"],"tech":["cloud"],"status":"model-ready"},
   {"id":"go.google-gloud-api","imports":["github.com/GoogleCloudPlatform/gcloud-golang"],"name":"Google Gloud API","repo":"GoogleCloudPlatform/gcloud-golang","tags":["gcloud"],"tech":["cloud"],"status":"model-ready"},
   {"id":"go.grpc","imports":["google.golang.org/grpc"],"name":"grpc","repo":"grpc/grpc-go","tags":["microservices"],"tech":["cloud"],"status":"model-ready"},
   {"id":"go.grpc-gateway","imports":["github.com/grpc-ecosystem/grpc-gateway"],"name":"grpc-gateway","repo":"grpc-ecosystem/grpc-gateway","tags":["microservices"],"tech":["cloud"],"status":"model-ready"},
+  {"id":"go.k3s","imports":["github.com/rancher/k3s"],"name":"k3s","repo":"rancher/k3s","tags":["kubernetes"],"tech":["cloud"],"status":"awaiting-model"},
```

Comment:
maybe lightweight-kubernetes for tags? looks like it's the whole point

Reply:
But also - are you sure it's a library? seems like a standalone product you would use to orchestrate your containers.

Reply:
you are right, will move it tools then